Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:14.04
+
+MAINTAINER Dali Freire
+
+USER root
```

Comment:
@dalifreire I think this is not needed, the Ubuntu base image starts you out as root afaik.